### PR TITLE
adding missing metadata for the language.

### PR DESCRIPTION
**Pull request by Janther**
Info taken from [linguist-languages](https://github.com/ikatyang/linguist-languages/blob/master/data/Solidity.json).

### CODE REVIEW

**Comment by Janther** [comment]
also we could have `linguist-languages` as a dependency and merge their json with a much leaner amount of data in our code.
```Javascript
solidityLang = require("linguist-languages/data/Solidity.json");

language = {
  ...solidityLang,
  since: "0.0.0",
  parsers: ['solidity-parse'],
  vscodeLanguageIds: ['solidity']
}
```

File: src/index.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 const print = require('./printer');
 
 // https://prettier.io/docs/en/plugins.html#languages
+const { languageId, ...rest } = linguistData;
 const languages = [
   {
-    extensions: ['.sol'],
-    name: 'Solidity',
+    linguistLanguageId: languageId,
+    ...rest,
```
Comment (mattiaerre) [inline src/index.js:14]: I would like to keep it explicit or else add what's in the `linguistData` object if possible

File: src/index.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+    since: '0.0.0',
```
Comment (mattiaerre) [inline src/index.js:15]: is this required?

File: src/index.js (hunk 1 above)
Comment (Janther) [reply src/index.js:14]: can do the object is very small.

File: src/index.js (hunk 2 above)
Comment (Janther) [reply src/index.js:15]: it's optional but am unsure if it's prettier's version, or our plugin's version

**Review by mattiaerre — APPROVED** [review]
thanks @Janther one very minor improvement would be to alpha order the properties but I'm happy to merge as is

**Comment by Janther** [comment]
> thanks @Janther one very minor improvement would be to alpha order the properties but I'm happy to merge as is
> 
> 

I'm keeping the properties like I got them from the linguist object for the moment we need to update this. Then we only need to copy paste the data on top.

**Pull request merged**